### PR TITLE
fix dpr_capitalprojects json loading

### PR DESCRIPTION
**Pull request by damonmcc**
## changes
* use header values when requesting DPR Capital Projects json
* abstract that step to a function
* use that function in other relevant `Scriptor.ingest()` functions

## notes
* this header `User-Agent` value was obtained by trial and error. tried the value found in my Chrome browser when downloading manually and it didn't work.

### CODE REVIEW

File: library/script/__init__.py [hunk 1]
```diff
@@ -7,3 +9,14 @@ def df_to_tempfile(df: pd.DataFrame) -> str:
     f = tempfile.NamedTemporaryFile(delete=False, suffix=".csv")
     df.to_csv(f, index=False)
     return f.name
+
+
+def get_json_content(url: str) -> str:
+    # use default headers to prevent "403 Client Error: Forbidden for url"
```
Comment (mbh329) [inline library/script/__init__.py:15]: Nice workaround to the 403 error. The user agent refers to the browser?

File: library/script/nypl_libraries.py [hunk 2]
```diff
@@ -12,8 +9,7 @@ def __init__(self, **kwargs):
 
     def ingest(self) -> pd.DataFrame:
         url = "https://refinery.nypl.org/api/nypl/locations/v1.0/locations"
-        content = requests.get(url).content
-        records = json.loads(content)["locations"]
+        records = get_json_content(url)["locations"]
```
Comment (mbh329) [inline library/script/nypl_libraries.py:12]: This succesfully ran in data-library

File: library/script/uscourts_courts.py [hunk 3]
```diff
@@ -18,8 +12,8 @@ def __init__(self, **kwargs):
         }
 
     def get_location(self, boro):
-        response = requests.get(self.sites.get(boro)).content
-        locations = json.loads(response).get("results").get("locations")
+        content = get_json_content(self.sites.get(boro))
```
Comment (mbh329) [inline library/script/uscourts_courts.py:15]: this successfully ran in data-library

File: library/script/__init__.py (hunk 1 above)
Comment (mbh329) [reply library/script/__init__.py:21]: Saw your comment in the PR about the user agent - doesn't seem super intuitive how to get the useragent

**Review by mbh329 — APPROVED** [review]

File: library/script/__init__.py (hunk 1 above)
Comment (damonmcc) [reply library/script/__init__.py:21]: unfortunately it isn't, and using the one produced by navigating to the site with my browser didn't work

the one that did work (and is in the code) is practically a random one people suggested using

**Pull request merged**